rust/read_json_file.rs

```diff
@@ -0,0 +1,24 @@
+//cargo-deps: json="0.12.4"
+
+use std::fs;
+use std::env;
+use json::JsonValue;
+extern crate json;
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    let filename = &args[1];
+
+    let json_string = fs::read_to_string(filename).unwrap();
+    let parsed_json = json::parse(&json_string).unwrap();
+
+    match parsed_json {
```

Comment:
Instead of using `match parse_json { ...` to ensure `parsed_json` is a `JsonArray`, let's do something like this:

```rust
let people = parsed_json.as_array().unwrap();
```